pages/docs/reference/python/middleware/encryption.mdx

```diff
@@ -0,0 +1,34 @@
+# Encryption middleware
+
+Encryption middleware provides end-to-end encryption for events, step output, and function output. **Only encrypted data is sent to Inngest servers**: encryption and decryption happen within your infrastructure.
+
+```py
+import inngest
+from inngest.experimental.encryption_middleware import EncryptionMiddleware
+
+inngest_client = inngest.Inngest(
+    app_id="my-app",
+    middleware=[EncryptionMiddleware.factory("my-secret-key")],
+)
+```
+
+The following data is encrypted by default:
+- The `encrypted` field in `event.data`.
+- `step.run` return values.
+- Function return values.
+
+## Changing the encrypted `event.data` field
+
+By default, `event.data.encrypted` is encrypted. All other fields are sent in plaintext. If you'd like to encrypt a different field, set the `event_encryption_field` parameter.
```

Comment:
```suggestion
By default, `event.data.encrypted` is encrypted. All other fields are sent in plaintext. To encrypt a different field, set the `event_encryption_field` parameter.
```

- more concise
- here we don't need to "cushion" the message because it's not a distressing news and we are not making a request to a user